spot

```diff
@@ -1,64 +1,59 @@
 #!/bin/bash
-if [[ $# != 1 ]]
-then
-	printf "Usage\n\tspot <URL>\n"
-	printf "\nExample\n\thttp://open.spotify.com/user/1248318865/playlist/7uKPSEwUQzx3c9EU032lU6\n"
-	exit 0
-fi
 
-input=$1
-keyword=spotify
-schemedurl=
-url=
+# enable shell script debugging
+# set -x
 
-matches() {
-    input="$1"
-    pattern="$2"
-    echo "$input" | grep -q "$pattern"
-}
-
-if matches $input $keyword
+# display help if incorrect usage or on --help
+if [[ $1 == "--help" ]]
```

Comment:
wouldn't it be better if we show the usage when there are zero arguments as well?